src/io/jenkins/infra/InfraConfig.groovy

```diff
@@ -25,17 +25,21 @@ class InfraConfig implements Serializable {
     return isJenkinsURLcontains('trusted.ci')
   }
 
+  Boolean isRelease() {
+    return isJenkinsURLcontains('release.ci')
+  }
+
   Boolean isInfra() {
     return isJenkinsURLcontains('infra.ci')
   }
 
   Boolean isRunningOnJenkinsInfra() {
-    return isCI() || isTrusted() || isInfra()
+    return isCI() || isTrusted() || isRelease() || isInfra()
   }
 
   // Returns the Docker registry hostname which this instance has credentials for
   String getDockerRegistryNamespace() {
-    if (isTrusted() || isInfra()) {
+    if (isTrusted() || isRelease() || isInfra()) {
```

Comment:
Not quite sure of this change, WDYT?

Reply:
this line should not be changed. Today, no Docker image is built on release.ci.jenkins.io.

If it ever happens, it would be `jenkinsci` namespace only, as the role of release.ci.jenkins.io is only to produce artifacts for the Jenkins users (e.g. Jenkins Core WAR, packages and eventuall Docker image) benefit.


`jenkinsciinfra` is a DockerHub namespace only aimed at the benefit of the Jenkins infrastructure: it's the role of infra.ci.jenkins.io.